docs/02-app/01-building-your-application/03-rendering/03-edge-and-nodejs-runtimes.mdx

```diff
@@ -27,7 +27,9 @@ There are many considerations to make when choosing a runtime. This table shows
 | Security                                                                                                                                            | Normal | High       | High             |
 | Latency                                                                                                                                             | Normal | Low        | Lowest           |
 | npm Packages                                                                                                                                        | All    | All        | A smaller subset |
-| Revalidation                                                                                                                                        | Yes    | Yes        | No               |
+| [Static Rendering](/docs/app/building-your-application/rendering/static-and-dynamic#static-rendering-default)                                       | Yes    | Yes        | No               |
```

Comment:
Why is Edge "No" for static rendering?

Reply:
From @ijjk's comment: https://vercel.slack.com/archives/C042LHPJ1NX/p1692668641730629?thread_ts=1692666204.185669&cid=C042LHPJ1NX